zipline/finance/metrics/__init__.py

```diff
@@ -107,4 +108,5 @@ def default_metrics():
         _ConstantCumulativeRiskMetric('treasury_period_return', 0.0),
         NumTradingDays(),
         PeriodLabel(),
+        ClassicRiskMetrics(),
```

Comment:
given that we can't actually access this in a backtest on zipline, we shouldn't put it in the default. Maybe we should register a default metrics set that includes this unioned with the default. What do you think?